src/Oscoin/Node.hs

```diff
@@ -84,6 +85,7 @@ withNode hConfig hNodeId hMempool hStateStore hBlockStore hProtocol hEval hConse
     bracket open close
   where
     open = do
+        hStateStore' <- newIoContentStore
```

Comment:
Perhaps `newContentStoreIO` is more fitting, as it's consistent to other similar APIs? (cfr. `newTVarIO` , etc)